config/initializers/geocoder.rb

```diff
@@ -11,7 +11,7 @@ def language=(_locale); end
 
 GEO_DATA_FILEPATH = Rails.root.join(IdentityConfig.store.geo_data_file_path).freeze
 
-if Rails.env.production? && File.exist?(GEO_DATA_FILEPATH)
+if File.exist?(GEO_DATA_FILEPATH)
```

Comment:
this is probably better served by a feature flag if we do want it to be configurable.